src/NodaTime/Duration.cs

```diff
@@ -1123,6 +1124,13 @@ public static Duration FromTimeSpan(TimeSpan timeSpan)
         public TimeSpan ToTimeSpan() => new TimeSpan(BclCompatibleTicks);
 
         #region XML serialization
+        /// <summary>
+        /// Adds the XML schema type describing the structure of the <see cref="Duration"/> XML serialization to the given <paramref name="xmlSchemaSet"/>.
+        /// </summary>
+        /// <param name="xmlSchemaSet">The XML schema set provided by <see cref="XmlSchemaExporter"/>.</param>
+        /// <returns>The qualified name of the schema type that was added to the <paramref name="xmlSchemaSet"/>.</returns>
+        public static XmlQualifiedName GetSchema(XmlSchemaSet xmlSchemaSet) => XmlSchemaDefinition.AddDurationSchemaType(xmlSchemaSet);
```

Comment:
Could the AddDurationSchemaType (etc) methods be extension methods, so we can just call `xmlSchemaSet.AddDurationSchemaType()`?

Reply:
It could be. I ended up not doing it to avoid having to add `using NodaTime.XmlSchemaDefinition;` in the 14 files with types implementing the IXmlSerializable interface but if you prefer, I can turn them into extension methods.

Reply:
Reasonable point about the using directive. I'm fine with keeping them as regular static methods - especially if it's all internal anyway.